src/modapi/server/modcreator.cpp

```diff
@@ -273,6 +370,12 @@ int CModAPI_ModCreator::Save(class IStorage *pStorage, const char *pFileName)
 		df.AddItem(MODAPI_MODITEMTYPE_LINESTYLE, i, sizeof(CModAPI_ModItem_LineStyle), &m_LineStyles[i]);
 	}
 	
```

Comment:
This doesn't work like this. Look at editor.cpp to see how they save image name. The structure ModItem contain only integers. In case of string, the integer store the size of the string. Then df.AddString is called (I'm not sure about the name) to store the string just after the structure.